packages/social-previews/src/mastodon-preview/previews.tsx

```diff
@@ -6,48 +6,59 @@ import { MastodonLinkPreview } from './link-preview';
 import { MastodonPostPreview } from './post-preview';
 import { MastodonPreviewProps } from './types';
 
-export type MastodonPreviewsProps = MastodonPreviewProps & SocialPreviewsBaseProps;
-
-export const MastodonPreviews: React.FC< MastodonPreviewsProps > = ( props ) => {
-	const { headingLevel, isSocialPost } = props;
+export type MastodonPreviewsProps = MastodonPreviewProps &
+	SocialPreviewsBaseProps & {
+		isSocialPost: boolean;
```

Comment:
Should we add this into the SocialPreviewsBaseProps? It's a flag that we'll need for all the previews.

Reply:
It'll get removed in the next PR anyway.

Reply:
Ok, so we're not going to have this property at all? It seems odd to move it in this PR then.

Reply:
The decision to remove the prop was not made at the time of this PR. I have restored it. Thanks